docs/introduction/syncing-overview.md

```diff
@@ -5,65 +5,71 @@ parent: Intro
 
 # Syncing Overview
 
-#### Background
+## Background
 
 Santa can be run and configured without a sync server. Doing so will enable an
 admin to configure rules with the `santactl rule` command. Using a sync server
-will enable an admin to configures rules and multiple other settings from the
+will enable an admin to configure rules and multiple other settings from the
 sync server itself. Santa was designed from the start with a sync server in
 mind. This allows an admin to easily configure and sync rules across a fleet of
 macOS systems. This document explains the syncing process.
 
-#### Flow of a full sync
+## Flow of a Full Sync
 
-NOTE: Synchronization is now performed by its own agent, the `santasyncservice`.
-The phases of synchronization described below still apply, but references to how
-the process starts is outdated. This will be updated soon.
+**NOTE:** Synchronization is now performed by its own daemon, the
+`santasyncservice`.
```

Comment:
It's probably been long enough that we can drop this now

Reply:
Removed.